Plugin~/Src/MeshSync/SceneGraph/msInstanceInfo.cpp

```diff
@@ -0,0 +1,69 @@
+#include "MeshSync/SceneGraph/msInstanceInfo.h"
+#include <MeshSync/msFoundation.h>
+#include <MeshSync/SceneGraph/msIdentifier.h>
+
+namespace ms {
+	void InstanceInfo::serialize(ostream& os)
```

Comment:
Using tabs with 8 spaces ?

Reply:
yes, lets change tabs to 4 spaces in the formatting PR, github seems to be converting 4 space tabs to 8 space tabs